app/cypress/e2e/cif/project-revision/create-first-project-revision.cy.js

```diff
@@ -79,6 +79,13 @@ describe("when creating a project, the project page", () => {
       "2020-02-02",
       20
     );
+    cy.findByText(/Total Project Value$/i)
+      .next()
+      .should("have.text", "$1,022.00"); // check that calculated value updates
+    cy.findByText(/Proponent's Share Percentage$/i)
+      .next()
+      .should("have.text", "78.27%"); // check that calculated value updates
```

Comment:
I'm not checking calculated values in the payments tracker because they're calculated from milestone form values, so they're not going to dynamically update on this page as formData changes.